packages/web-components/fast-foundation/src/data-grid/data-grid-row.template.ts

```diff
@@ -14,7 +14,7 @@ function createCellItemTemplate(prefix: string): ViewTemplate {
 
 function createHeaderCellItemTemplate(prefix: string): ViewTemplate {
     return html`
-    <${prefix}-data-grid-cell
+    <${prefix}-data-grid-header-cell
```

Comment:
This should be the opposite, remove the "header" from the closing tag.  I'm surprised this didn't cause issues but in our samples the first tag "wins".